### PR TITLE
Better: Add capabilities_supported for contents in API and webhooks. RD-23140

### PR DESCRIPTION
**Pull request by ylecuyer**
We are adding capabilities_supported for all contents to the API and webhooks so we reflect the info here

### CODE REVIEW

File: specs/engage-digital_openapi3.yaml [hunk 1]
```diff
@@ -7122,6 +7122,11 @@ components:
         updated_at:
           format: date-time
           type: string
+        capabilities_supported:
```
Comment (ylecuyer) [inline specs/engage-digital_openapi3.yaml:7125]: [README.md](https://github.com/ringcentral/engage-digital-api-docs/files/9302120/README.md)

File: specs/engage-digital_openapi3.yaml [hunk 2]
```diff
@@ -136,7 +136,7 @@ paths:
           name: topologyId
           required: true
           schema:
-            type: string
+            type: integer
```
Comment (CedricBm) [inline specs/engage-digital_openapi3.yaml:139]: But it is a string, no?

File: specs/engage-digital_openapi3.yaml [hunk 3]
```diff
@@ -126,7 +126,7 @@ paths:
   '/topologies/{topologyId}':
     put:
       description: >-
-        This method updates an existing topology from given attributes and renders it in case of success.
+        This method an existing topology from given attributes and renders it in case of success.
```
Comment (CedricBm) [inline specs/engage-digital_openapi3.yaml:129]: Wrong update?

File: docs/webhooks/events.md [hunk 4]
```diff
@@ -223,6 +223,7 @@ Property | Value | Description
 `synchronization_status` | String | For content.exported and content.update_exported event, the status of the synchronization (successful or unsuccessful).
 `thread_id` | String | ID of the thread that owns the content
 `thread_title` | String| Title of the thread that owns the content.
+`capabilities_supported` | Array of Strings | List the kinds of sturctured messages that can be used to reply to this kind of message.
```
Comment (ylecuyer) [inline docs/webhooks/events.md:226]: Screenshot_2022-08-11_11-42-50

File: specs/engage-digital_openapi3.yaml (hunk 2 above)
Comment (ylecuyer) [reply specs/engage-digital_openapi3.yaml]: yes it was a test for something else, you should wait for the PR to be assigned and ready before reviewing ^^

File: docs/webhooks/events.md (hunk 4 above)
Comment (kalilz4485) [inline docs/webhooks/events.md:226]:
```suggestion
`capabilities_supported` | Array of Strings | List the kinds of structured messages that can be used to reply to this kind of message.
```

File: specs/engage-digital_openapi3.yaml [hunk 5]
```diff
@@ -7122,6 +7122,11 @@ components:
         updated_at:
           format: date-time
           type: string
+        capabilities_supported:
+          description: lists the kinds of sturctured messages that can be used to reply to this kind of message
```
Comment (kalilz4485) [inline specs/engage-digital_openapi3.yaml:7126]:
```suggestion
          description: lists the types of structured messages that can be used to reply to this type of message

```

**Comment by CedricBm** [comment]
If I remember correctly, adding a field in the response payload doesn't change the postman file, so lgtm after Cyril's suggestions.

**Comment by ylecuyer** [comment]
> If I remember correctly, adding a field in the response payload doesn't change the postman file, so lgtm after Cyril's suggestions.

Now there is a test checking that the postman file is up to date, so it would fail the build if an update is needed

**Pull request merged**